### PR TITLE
feat: add /key-blocks/:hash_or_kbi endpoint with mbs/txs count

### PR DESCRIPTION
**Pull request by sborrazas**
refs #888

### CODE REVIEW

**Review by jyeshe — COMMENTED** [review]
Some minors...

File: test/ae_mdw_web/controllers/block_controller_test.exs [hunk 1]
```diff
@@ -98,6 +98,91 @@ defmodule AeMdwWeb.BlockControllerTest do
...
+    end
+
+    test "it returns 404 when not found", %{conn: conn, store: store} do
+      decoded_hash = TS.key_block_hash(0)
+      encoded_hash = :aeser_api_encoder.encode(:key_block_hash, decoded_hash)
+      error_msg = "not found: #{encoded_hash}"
+
+      with_mocks [
+        {:aec_chain, [], [get_block: fn ^decoded_hash -> :error end]}
+      ] do
```
Comment (jyeshe) [inline test/ae_mdw_web/controllers/block_controller_test.exs:176]: It is not needed (the ci database won't have it)

File: lib/ae_mdw/blocks.ex [hunk 2]
```diff
@@ -121,34 +152,34 @@ defmodule AeMdw.Blocks do
...
-      last_mbi =
-        case State.prev(state, @table, {gen + 1, -1}) do
-          {:ok, {^gen, mbi}} -> mbi + 1
-          {:ok, _block_index} -> 0
-          :none -> 0
-        end
+  defp render_key_blocks(state, range), do: Enum.map(range, &render_key_block(state, &1))
+
+  defp render_key_block(state, gen) do
+    last_mbi =
```
[the remark this answers is not in the document]
Comment (sborrazas) [reply lib/ae_mdw/blocks.ex]: Done

File: test/ae_mdw_web/controllers/block_controller_test.exs [hunk 3]
```diff
@@ -98,6 +98,91 @@ defmodule AeMdwWeb.BlockControllerTest do
...
+         [
+           height: fn :header -> kbi end,
+           serialize_for_client: fn :header, :micro -> %{height: kbi} end,
+           type: fn :header -> :key end
+         ]},
+        {AeMdw.Node.Db, [], [prev_block_type: fn :header -> :micro end]}
+      ] do
+        assert %{
+                 "height" => ^kbi,
+                 "transactions_count" => 6
```
[the remark this answers is not in the document]
Comment (sborrazas) [reply test/ae_mdw_web/controllers/block_controller_test.exs:164]: Done

File: test/ae_mdw_web/controllers/block_controller_test.exs (hunk 1 above)
Comment (sborrazas) [reply test/ae_mdw_web/controllers/block_controller_test.exs]: Done

**Review by thepiwo — APPROVED** [review]

**Pull request merged**